webpack/assets/javascripts/react_app/components/AuthSource/Welcome.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { foremanUrl, getManualURL } from '../../common/helpers';
+
+export const WelcomeAuthSource = ({ canCreate }) => {
+  const content1 = __(`The authentication process currently requires an LDAP provider, such as <em>FreeIPA</em>, <em>OpenLDAP</em> or <em>Microsoft's Active Directory</em>.
+  <a href=${getManualURL(
+    '4.1.1LDAPAuthentication'
+  )}>Learn more about LDAP authentication in the documentation.</a>`);
+  const content2 = __(`Foreman can use External service for user information and authentication. <br/>
+  <a href=${getManualURL(
+    '5.7ExternalAuthentication'
+  )}>Learn more about External authentication in the documentation.</a>`);
```

Comment:
This layout looks a bit odd... I'm not sure if eslint will permit it, but something like this would probably be easier to read:
```suggestion
  const content1 = __(`The authentication process currently requires an LDAP provider,
     such as <em>FreeIPA</em>, <em>OpenLDAP</em> or <em>Microsoft's Active Directory</em>.
    <a href=${getManualURL('4.1.1LDAPAuthentication')}>Learn more about LDAP authentication in the documentation.</a>`);
  const content2 = __(`Foreman can use External service for user information and authentication. <br/>
    <a href=${getManualURL('5.7ExternalAuthentication')}>Learn more about External authentication in the documentation.</a>`);
```
Although you don't need to include all of the strings here in the dangerous html, it's just the one sentence that includes the `<em>` tags here, since the links are displayed on their own lines and not as part of the same sentence.